### PR TITLE
AC Automated Test - LRAC-8504 | master 

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRAC-8504

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverviewAttributeBreakdown.testcase [hunk 1]
```diff
@@ -32,9 +32,91 @@ definition {
 	@ignore = "true"
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverviewAttributeBreakdown.testcase:32]: hi @epark18 , could you please remove ignore from this test?

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**